pkg/report/output/dataflow/types/components.go

```diff
@@ -1,7 +1,7 @@
 package types
 
 type Component struct {
-	Name      string
+	Uuid      string
```

Comment:
Currently in the codebase, we either have `UUID` or `uuid` (for private). Suggestion to prefer `UUID` for consistency here. 

```suggestion
	UUID      string
```